tensorflow/python/summary/summary.py

```diff
@@ -104,6 +104,7 @@ def scalar(name, tensor, collections=None):
     tensor: A real numeric Tensor containing a single value.
     collections: Optional list of graph collections keys. The new summary op is
       added to these collections. Defaults to `[GraphKeys.SUMMARIES]`.
+    prefix: Optional string tensor
```

Comment:
Optional string tensor. If provided, it will be a prefix to the summary's tag in TensorBoard.